tests/rptest/tests/rpk_cluster_test.py

```diff
@@ -343,3 +343,34 @@ def test_upload_cluster_license_error(self):
                 tf.seek(0)
 
                 self._rpk.license_set(tf.name)
+
+    @cluster(num_nodes=3)
+    def test_expired_evaluation_period(self):
+        """
+        Test that once the eval period license expires, it is not observable
+        """
+        self.logger.debug("Ensure the eval period license is expired")
+        self.redpanda.set_environment(
+            dict(__REDPANDA_DISABLE_BUILTIN_TRIAL_LICENSE='1'))
+        self.redpanda.rolling_restart_nodes(self.redpanda.nodes,
+                                            use_maintenance_mode=False)
+        self.redpanda.wait_until(
+            self.redpanda.healthy,
+            timeout_sec=15,
+            backoff_sec=1,
+            err_msg="The cluster failed to stabilize after restart")
+
+        wait_until(
+            lambda: self._get_license_expiry() == -1,
```

Comment:
does the metric just returns `-1` when the license in expired?

Reply:
> does the metric just returns `-1` when the license in expired?

It returns -1 if there is not a licence, the license is determined by calling this function.